docs/Technical Reference.md

```diff
@@ -141,69 +141,55 @@ A **bucket config**, meanwhile, is a JSON object that describes metadata associa
 
 A bucket config can be included inline in a federation, or it can be a standalone JSON file that is linked from a federation.
 
-## Adding users to a catalog using AWS Console
+## Preparing an AWS Role for use with T4
 
-These instructions document how to add users to your catalog instance using the AWS Console.
+These instructions document how to set up an existing role for use with T4. If the role you want to use doesn't exist yet, create it now.
```

Comment:
Can you recommend a good default?

Reply:
not really

Reply:
I'm going to test `S3FullAccess` and see how that goes